source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/user-dashboard/groupDropdown.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { Dropdown, Checkbox } from 'semantic-ui-react';
+
+
+export interface IProps {
+  items: Array<{ id: string, value: string}>;
+  selected: Array<{id: string, value: string}>;
+}
+
+export interface IState {
+  items: Array<{ id: string, value: string }>;
+  selected: Array<{ id: string, value: string }>;
+}
+
+class GroupDropdown extends React.Component<IProps, IState> {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      items: props.items,
+      selected: props.selected
+    }
+
+  }
+
+
+  selectedItems = () => {
+    const { selected, items } = this.state;
+
+    if (selected && selected.length == 0) {
+      return '-';
```

Comment:
what does '-' mean?

Reply:
maybe we should say something else here? or is this in the deisgns?

Reply:
The design shows a - when no group is assigned, we could ask Josh if this can be change

Reply:
all good